ipn/ipnlocal/local.go

```diff
@@ -453,6 +453,24 @@ func (b *LocalBackend) updateStatus(sb *ipnstate.StatusBuilder, extraLocked func
 	}
 }
 
+// exitNodeIP returns the exit node IP from and netMap.
+func (b *LocalBackend) exitNodeIP() (ip netaddr.IP) {
```

Comment:
no need to name the result

see https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
I've seen named results in other parts of the codebase so confirming our approach moving forward is to avoid this?